.github/workflows/spellcheck.yml

```
@@ -0,0 +1,20 @@
+name: Spellcheck
+
+on: [push, pull_request]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  spellcheck:
+    name: Spelling
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        ref: ${{ github.event.pull_request.head.sha }}
+        show-progress: false
+    - uses: codespell-project/actions-codespell@master
+      with:
```

```suggestion
      with:
      builtin: clear,en-GB_to_en-US
```
This should specify the language options.

---

```suggestion
      with:
        builtin: clear,en-GB_to_en-US
```